yarn-project/aztec-sandbox/src/sandbox.ts

```diff
@@ -103,6 +103,8 @@ async function deployContractsToL1(aztecNodeConfig: AztecNodeConfig, hdAccount:
     l1Contracts.l1ContractAddresses.contractDeploymentEmitterAddress;
   aztecNodeConfig.l1Contracts.inboxAddress = l1Contracts.l1ContractAddresses.inboxAddress;
   aztecNodeConfig.l1Contracts.registryAddress = l1Contracts.l1ContractAddresses.registryAddress;
+  aztecNodeConfig.l1Contracts.outboxAddress = l1Contracts.l1ContractAddresses.outboxAddress;
+  aztecNodeConfig.l1Contracts.decoderHelperAddress = l1Contracts.l1ContractAddresses.decoderHelperAddress;
```

Comment:
The return value is never used since we read the info from the config so we could replace the last 11 lines with:

` 

    ...
    aztecNodeConfig.l1Contracts = (
        await waitThenDeploy(aztecNodeConfig, () =>
          deployL1Contracts(aztecNodeConfig.rpcUrl, hdAccount, localAnvil, logger, l1Artifacts),
        )
      ).l1ContractAddresses;
`